app/helpers/application_helper.rb

```diff
@@ -76,59 +77,58 @@ def state_thumbnail(name)
 
   def us_states
     {
-      'Alabama'              => 'AL',
-      'Alaska'               => 'AK',
-      'Arizona'              => 'AZ',
-      'Arkansas'             => 'AR',
-      'California'           => 'CA',
-      'Colorado'             => 'CO',
-      'Connecticut'          => 'CT',
-      'Delaware'             => 'DE',
+      'Alabama' => 'AL',
```

Comment:
you updated the hash style throughout the rest of the app, may be worth updating it here for consistency.

Reply:
I believe this is acceptable because the key is not a symbol (https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/HashSyntax)
how would you like this changed